framework/packages/abstract-core/src/objects/account/account_id.rs

```diff
@@ -76,6 +76,43 @@ impl AccountId {
     }
 }
 
+impl TryFrom<&str> for AccountId {
+    type Error = AbstractError;
+
+    fn try_from(value: &str) -> Result<Self, Self::Error> {
+        let mut val_iter = value.rsplitn(2, '-');
```

Comment:
let's use rsplit_once instead

Reply:
Here's me thinking I know the `std` crate xd